app/sidekiq/simple_forms_api/notification/send_notification_email_job.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+module SimpleFormsApi
+  module Notification
+    class SendNotificationEmailJob
+      include Sidekiq::Job
+
+      sidekiq_options retry: 10
+
+      HOUR_TO_SEND_NOTIFICATIONS = 9
+
+      attr_reader :notification_type, :config, :form_number, :user_account
+
+      def perform(notification_type:, form_submission_attempt:, user_account:)
+        @notification_type = notification_type
+        @user_account = user_account
+        form_submission = form_submission_attempt.form_submission
+        @form_number = V1::UploadsController::FORM_NUMBER_MAP[form_submission.form_type]
```

Comment:
Will the form number always be in the `FORM_NUMBER_MAP`? Is it possible for `form_submission.form_type` to be invalid?

Reply:
Good point. It shouldn't be a problem but in case it is I've moved the rescue up to this `perform` method so that we'll increment StatsD in such cases. Does that sound like a good resolution?